keep/api/core/db.py

```diff
@@ -2921,25 +2922,16 @@ def is_alert_assigned_to_incident(
     with Session(engine) as session:
         assigned = session.exec(
             select(LastAlertToIncident)
+            .join(Incident, LastAlertToIncident.incident_id == Incident.id)
```

Comment:
is this a bug now?

Reply:
Not actually. If incident is deleted we treat it as Alert is not assigned to the incident anymore